src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -2416,7 +2416,7 @@ void DistributePlantLoad(EnergyPlusData &state,
     {
         Real64 plant_capacity_to_this_point;
         Real64 largest_min_plr_to_this_point;
-        LoadPLRPoint(Real64 capacity, Real64 plr)
+        LoadPLRPoint(Real64 capacity, Real64 plr) : plant_capacity_to_this_point(), largest_min_plr_to_this_point()
         {
             plant_capacity_to_this_point = capacity;
             largest_min_plr_to_this_point = plr;
```

Comment:
Performance suggestion:

    [src/EnergyPlus/PlantCondLoopOperation.cc:2421]:(performance),[useInitializationList],Variable 'plant_capacity_to_this_point' is assigned in constructor body. Consider performing initialization in initialization list.
    [src/EnergyPlus/PlantCondLoopOperation.cc:2422]:(performance),[useInitializationList],Variable 'largest_min_plr_to_this_point' is assigned in constructor body. Consider performing initialization in initialization list.

Reply:
This change did not get rid of the performance warnings. Since I wasn't sure what this message was suggesting, this must not be the correct fix.

Reply:
It wants you to do this:

```cpp
LoadPLRPoint(Real64 capacity, Real64 plr) : plant_capacity_to_this_point(capacity), largest_min_plr_to_this_point(plr) {}
```